python-package/xgboost/core.py

```diff
@@ -1579,9 +1611,17 @@ def predict(self,
                 preds = preds.reshape(nrow, chunk_size)
         return preds
 
-    def inplace_predict(self, data, iteration_range=(0, 0),
-                        predict_type='value', missing=np.nan):
-        '''Run prediction in-place, Unlike ``predict`` method, inplace prediction does
+    def inplace_predict(
+        self,
+        data,
+        iteration_range: Tuple[int, int] = (0, 0),
+        predict_type: str = "value",
+        missing: float = np.nan,
+        validate_features: bool = True,
+        base_margin: Any = None,
+        strict_shape: bool = False
```

Comment:
This parameter may needs more thought.  It's meant to be a placeholder for https://github.com/dmlc/xgboost/pull/6638 .  It affects only the leaf prediction.